netlify.toml

```diff
@@ -0,0 +1,24 @@
+# Netlify build instructions 
+[build]
+    command = "./hack/build-gitbook.sh"
+    publish = "docs/book/_book"
+
+# Standard Netlify redirects
+[[redirects]]
+    from = "https://kubernetes-sigs-cluster-api.netlify.com/*"
+    to = "https://cluster-api.sigs.k8s.io/:splat"
```

Comment:
This seems like it would lead to a broken config prior to https://github.com/kubernetes/k8s.io/pull/267 landing. Just making sure that is the expected behavior, rather than an oversight.

Reply:
That is correct. I have disabled redirects temporarily so that others can validate the generated site. I plan to add them back as soon as the domain exists... WDYT?